src/main/groovy/com/tdc/bloop/models/StringResponse.groovy

```diff
@@ -4,5 +4,14 @@ package com.tdc.bloop.models
  * Created by ian.sanchez on 5/31/17.
  */
 class StringResponse {
-    String text
```

Comment:
@iangabrielsanchez 
Setter and Getter global variables should always be private right?
Or is that just a Java thing. Not Groovy?